### PR TITLE
Softwareeconomics

### PR DESCRIPTION
**Pull request by egedebruin**
I created and finished exercise 3
Please take a good look, also for the english xp

### CODE REVIEW

File: deliverables/sprint3/Exercise3.tex [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Of the Visual Basic projects, the most of those ended up in a good practice project. But it is not so interesting because there were only 6 Visual Basic projects, which is already a small amount of projects within the 352 projects analyzed in the paper. Furthermore, are on average less complex than other projects, therefore end up as a good practice project more often. 
+
+\subsection{Exercise 3.3 - Good/Bad factors}
+\begin{enumerate}
+\item Planned testing \hfill \\
+This belongs to a good practice. It is important to not test at the last moment. It needs planning to make sure that the testing is not only done when the schedule gets tight. It is also important for a project to plan test cases before the coding starts. This all makes sure that members in a project do not waste a lot of time in fixing many tests at the same time. Planning the tests will save a lot of time.
+\item Code review \hfill \\
+This belongs to a good practice. A lot of problems are eliminated earlier when a project team regularly reviews each others code. It is important to review to code on bugs, design, architecture and everything to make the code better. This may be more effective than testing.
```
Comment (jochem725) [inline deliverables/sprint3/Exercise3.tex]: "review to code on bugs" should be "review the code on bugs"

**Comment by jochem725** [comment]
Looks good, only I'm not sure the sentence "it costs a lot" is correct english. It appears several times in the document. I think "it takes a lot" would fit better.

**Comment by doriandekoning** [comment]
Seems fine, I think you can say "it costs a lot" but its not a very formal, nor a very nice way to express it.

**Comment by egedebruin** [comment]
I changed it according to your comments.

**Comment by doriandekoning** [comment]
Great!

**Pull request merged**